.cicd/platforms/pinned/oracle_linux-8.3-pinned.dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM oraclelinux:8.3
```

Comment:
For that reason explained in the comment above, we might even consider being less specific here. You can see `yum update` is the very first thing we do in the dockerfile. So, similarly, calling out `oraclelinux:8.3` is misleading.